packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/src/models/models.ts

```diff
@@ -47,6 +73,15 @@ export interface TrackedResource extends Resource {
   tags?: Record<string, string>;
 }
 
+export function trackedResourceSerializer(
```

Comment:
I wonder if we should have an independant file to put those serializer, they are not suppose to be exposed to external customers right? Another concern I have is If we release mpg preview with it, then remove it in the later iteration, it will be a breaking change right?

Reply:
I noticed that you have replied somewhere else.